packages/db/src/types/execution.ts

```diff
@@ -1,10 +1,20 @@
+import { ExecutionRecordStatus } from '@cumulus/types/api/executions';
+
 export interface PostgresExecution {
   arn: string,
-  status: string,
   async_operation_cumulus_id?: number,
   collection_cumulus_id?: number,
   parent_cumulus_id?: number,
   cumulus_version?: string,
+  url?: string,
+  status?: ExecutionRecordStatus,
```

Comment:
`status` is not nullable in the PG schema, so it should have to be set even before the record was saved to the database. So it should be required in the interface, I believe

Reply:
Oh yeah, I missed that. Updated to a required param.